spring-batch-infrastructure/src/main/java/org/springframework/batch/item/json/GsonJsonObjectMarshaller.java

```diff
@@ -30,9 +30,18 @@
 
 	private Gson gson = new Gson();
```

Comment:
According to the improvement suggested in this PR, there shouldn't be an inline instantiation of `Gson` here (similar to what has been done in other classes in this change set). I will update that on merge.